sdks/java/io/elasticsearch/build.gradle

```diff
@@ -27,7 +27,7 @@ dependencies {
   shadow project(path: ":beam-sdks-java-core", configuration: "shadow")
   shadow library.java.jackson_databind
   shadow library.java.jackson_annotations
-  shadow "org.elasticsearch.client:elasticsearch-rest-client:5.6.3"
+  shadow "org.elasticsearch.client:elasticsearch-rest-client:6.3.2"
```

Comment:
Nice to see that REST client v 6.3.2 is still retro-compatible with ES2 and ES5. I chose this low level client over the other high level client in the first place to have the common set of production libraries between all the versions. Only test dependencies differ